package.json

```diff
@@ -41,26 +41,53 @@
 		"ps",
 		"proc"
 	],
+	"babel": {
+		"presets": [
+			"@babel/preset-react"
```

Comment:
Make sure that Babel does not transform ESM

Reply:
> Make sure that Babel does not transform ESM

I was trying to use babel here only to use JSX.
(If possible, using [import-jsx](https://github.com/vadimdemedes/import-jsx) would be better, but [does not support ESM yet](https://github.com/vadimdemedes/import-jsx/issues/15))
Do we need to transform ESM to CJS or other tasks?

Reply:
> Do we need to transform ESM to CJS or other tasks?

Please re-read my comment. We do **not** want to transpile ESM to CJS. I think Babel does it by default.

https://babeljs.io/docs/en/options#targetsesmodules

Reply:
> Please re-read my comment. We do **not** want to transpile ESM to CJS.

I'm sorry for the misunderstanding.

> I think Babel does it by default.

I used Babel only for using JSX here, so other transpiling configs not included.
You can see Babel is transpiling the source code to ESM itself under the `distribution` folder.

Reply:
> You can see Babel is transpiling the source code to ESM itself under the distribution folder.

I see that, but that contradicts their own docs:

> When no targets are specified: Babel will assume you are targeting the oldest browsers possible. 

- https://babeljs.io/docs/en/options#no-targets

Reply:
> I see that, but that contradicts their own docs:

I posted a related question in the Babel discussions, and I got the below answer.

https://github.com/babel/babel/discussions/14533#discussioncomment-2700315